src/actions/index.js

```diff
@@ -0,0 +1,59 @@
+import {
+    getStudent,
+    getStudentTotalScore
```

Comment:
We can reduce it on the client. I think its better to get all records and then with the magic of selectors retrieve anything from it